plugins/draw-tools.js

```diff
@@ -654,11 +681,112 @@ window.plugin.drawTools.boot = function() {
+// EXPORT (POLYS AS LINES)
+// ---------------------------------------------------------------------------------
+window.plugin.drawTools.getDrawAsLines = function(){
+  var rawDraw = JSON.parse(window.localStorage[window.plugin.drawTools.KEY_STORAGE]);
+  var draw = [];
+
+  for(i in rawDraw){
+    var elemDraw = rawDraw[i];
+
+    if(elemDraw.type === 'polygon'){
+      var convElemDraw = {};
+      convElemDraw.color = elemDraw.color;
+      convElemDraw.type = 'polyline';
+      convElemDraw.latLngs = [];
+
+      var v = elemDraw.latLngs.length;
+      for(j in elemDraw.latLngs){
+        var ll = elemDraw.latLngs[j];
+        convElemDraw.latLngs.push(ll);
+      }
+      convElemDraw.latLngs.push(elemDraw.latLngs[0]);
+
+      draw.push(convElemDraw);
+    }else{
+      draw.push(elemDraw);
+    }
+  }
+
+  return JSON.stringify(draw);
+}
+
+// ---------------------------------------------------------------------------------
+// EMPTY POLYGONS (EMPTY DRAWN FIELDS)
+// ---------------------------------------------------------------------------------
+window.plugin.drawTools.edf = {};
+window.plugin.drawTools.edf.storage = {};
+window.plugin.drawTools.edf.draw = {};
+window.plugin.drawTools.edf.obj = {};
+window.plugin.drawTools.edf.action = {};
+window.plugin.drawTools.edf.ui = {};
+
+window.plugin.drawTools.edf.boot = function(){
+  window.addHook('pluginDrawTools', window.plugin.drawTools.edf.hookManagement);
+  window.addHook('iitcLoaded', function(){
+    if(window.plugin.drawTools.edf.obj.status){
+      window.plugin.drawTools.edf.draw.toggleOpacityOpt();
+      window.plugin.drawTools.edf.draw.clearAndDraw();
+    }
+  });
+};
+
+window.plugin.drawTools.edf.obj.status = false;
+window.plugin.drawTools.edf.obj.toggle = function(){
+  var status = window.plugin.drawTools.edf.obj.status;
+  status = Boolean(!status);
+  window.plugin.drawTools.edf.obj.status = status;
+}
+window.plugin.drawTools.edf.draw.toggleOpacityOpt = function(){
+  if(window.plugin.drawTools.edf.obj.status){
+    window.plugin.drawTools.polygonOptions.fillOpacity = 0.0;
+  }else{
+    window.plugin.drawTools.polygonOptions.fillOpacity = 0.2;
+  }
+}
+window.plugin.drawTools.edf.draw.clearAndDraw = function(){
+  window.plugin.drawTools.drawnItems.clearLayers();
+  window.plugin.drawTools.load();
+  console.log('DRAWTOOLS: reset all drawn items');
+}
+
+window.plugin.drawTools.edf.action.toggle = function(){
+  window.plugin.drawTools.edf.obj.toggle();
+  window.plugin.drawTools.edf.draw.toggleOpacityOpt();
+  window.plugin.drawTools.edf.draw.clearAndDraw();
+}
+
+window.plugin.drawTools.edf.hookManagement = function(data){
+  if(data.event === 'openOpt'){
+  }else if(data.event == 'layerCreated'){
+    if(window.plugin.drawTools.edf.obj.status){
+      //    window.plugin.drawTools.edf.draw.toggleOpacityOpt(); // disabled by Zaso
+      window.plugin.drawTools.edf.draw.clearAndDraw();
+    }
+  }
+}
+
+window.plugin.drawTools.setupCSS = function(){
```

Comment:
this function is never called

Reply:
It used to be executed from setup() in Drawtools-Plus. For reference and for that I can currently not oversee what other plugins might be using the CSS-changes the code is left in place but might be removed in near future.

Reply:
@MysticJay Please explain your reply.
Because current code is wrong: you either need that css (then you should apply it somehow), either you do not need it (then it should be removed)

Reply:
Unused code removed from PR as not related.